packages/hasura-storage-js/src/utils/types.ts

```diff
@@ -1,3 +1,11 @@
+import { HasuraAuthClient } from '@nhost/hasura-auth-js'
+
+// * Avoid circular references and broken links in docusaurus generated docs
+export interface INhostClient {
```

Comment:
Can we call it `NhostClientReturnType` or something similar? We should try to avoid the `I` prefix, because it doesn't describe the interfaces purpose properly.